test/server/test-run-commands-test.js

```diff
@@ -3685,3 +3722,18 @@ describe('Test run commands', () => {
         });
     });
 });
+
+describe('Test run screenshot', () => {
```

Comment:
In this tests file we check command creation logic. Let's find a more suitable place for this test.

Reply:
I think there is no need for this test after @Bayheck fixed the test 'Should take a screenshot on fail'. What do you think?

Reply:
I think so too, I've already removed this test from PR